docs/user-guide/rug/generators.md

```diff
@@ -16,46 +16,135 @@ Let's look more closely at what makes a project a Rug generator.
 
 ## Anatomy of a Generator
 
-Suppose we have a model project for creating new documentation
-projects based on the [MkDocs][mkdocs] static site generator.  The
-contents of the model project are the following.
+Suppose we have a model project our team clones to quickly get the skeleton of a 
+[Spring Bot Rest Service][springrest].  The contents of the model project are 
+the following.
 
-[mkdocs]: http://www.mkdocs.org/
+[springrest]: https://spring.io/guides/gs/rest-service/
 
 ```console
-mkdocs-project
+spring-boot-rest-basic
+    ├── .gitignore
+    ├── pom.xml
     ├── README.md
-    ├── docs
-    │   └── index.md
-    ├── mkdocs.yml
-    └── requirements.txt
+    ├── src
+    │   ├── main
+    │   │   ├── java
+    │   │   │   └── com
+    │   │   │       └── company
+    │   │   │           ├── HomeController.java
+    │   │   │           ├── MyRestServiceApplication.java
+    │   │   │           └── MyRestServiceConfiguration.java
+    │   │   └── resources
+    │   │       ├── application.properties
+    │   │       └── logback.xml
+    │   └── test
+    │       └── java
+    │           └── com
+    │               └── company
+    │                   ├── MyRestServiceApplicationTests.java
+    │                   ├── MyRestServiceOutOfContainerIntegrationTests.java
+    │                   └── MyRestServiceWebIntegrationTests.java
+    └── .travis.yml
```

Comment:
The default has a `.travis.yml`? We may want to leave that out and add it later with an editor?